grails-app/controllers/org/pih/warehouse/inventory/InventoryController.groovy

```diff
@@ -138,6 +138,8 @@ class InventoryController {
         command.location = command?.location ?: Location.get(session.warehouse.id)
         def category = params.categoryId ? Category.get(params.categoryId) : productService.getRootCategory()
         command.category = category?.id ? category : null
+        command.catalogs = params.catalogs ? command.catalogs : null
+        command.tags = params.tags ? command.tags : null
```

Comment:
it was needed to be added, because when you click "Delete all" (x) on this ajax select, the request is proceeding with for example: `catalogs=&....`, so an Array with one null element is passed, which made the response having no results - it got fixed just after another fetch, so if `params.catalogs` is empty string, we want to make the `command.catalogs` as `null`, not `[null]`